docs/book/src/user/quick-start.md

```diff
@@ -795,13 +853,13 @@ Please visit the [IBM Cloud provider] for more information.
 {{#/tab }}
 {{#tab Kubevirt}}
 
-A ClusterAPI compatible image must be available in your Kubevirt image library. For instructions on how to build a compatible image
-see [image-builder](https://image-builder.sigs.k8s.io/capi/capi.html).
-
-To see all required Kubevirt environment variables execute:
 ```bash
-clusterctl generate cluster --infrastructure kubevirt --list-variables capi-quickstart
+export CAPK_GUEST_K8S_VERSION="${CAPK_GUEST_K8S_VERSION:-v1.23.10}"
+export CRI_PATH="${CRI_PATH:-/var/run/containerd/containerd.sock}"
+export NODE_VM_IMAGE_TEMPLATE="${NODE_VM_IMAGE_TEMPLATE:-quay.io/capk/ubuntu-2004-container-disk:${CAPK_GUEST_K8S_VERSION}}"
+export IMAGE_REPO="${IMAGE_REPO:-k8s.gcr.io}"
```

Comment:
fyi. The new default with v1.25 is registry.k8s.io (there's a redirect right now, so both serve the same images)

Please consider using the new registry

Reply:
The latest image is for 1.23

@davidvossel - are we planing to introduce an image for 1.25?

Reply:
For some reason this is just not working and I can't debug it. I will leave the original value of `k8s.gcr.io`.

Reply:
Okay, please follow-up on this. Kubernetes is literally running out of money with the old registry. So we're trying everything we can going forward to use the new registry as it's cheaper.